Charon/filetypes/GCodeFile.py

```diff
@@ -22,6 +22,7 @@ class GCodeFile(FileInterface):
     mime_type = "text/x-gcode"
 
     MaximumHeaderLength = 100
+    SkipHeaderValidation = False
```

Comment:
In general, double negatives are something you may want to avoid. Better to introduce a setting `ValidateHeader` with a default of `True`.

Reply:
No particularly strong feelings, if it's something you've considered already. I do see your point :)